components/modal/modal-footer.component.ts

```diff
@@ -69,7 +72,9 @@ import { ModalButtonOptions, ModalOptions } from './modal-types';
   host: {
     class: 'ant-modal-footer'
   },
-  changeDetection: ChangeDetectionStrategy.Default
+  changeDetection: ChangeDetectionStrategy.Default,
+  imports: [NgIf, NzOutletModule, NgForOf, NzButtonModule],
```

Comment:
suggestion: you can import NgFor instead ngForOf

Reply:
Yeah you are right. changes been made.
But I find it a bit confusing as why NgForOf is suggested to be imported instead of NgFor...

Reply:
> Yeah you are right. changes been made. But I find it a bit confusing as why NgForOf is suggested to be imported instead of NgFor...

Yeah that’s something with your ide, don’t know which one you used and also the version 

but If you take a look at this link 

https://github.com/angular/angular/blob/main/packages/common/src/directives/ng_for_of.ts

You can see that NgForOf class is exported as NgFor